### PR TITLE
Created Script for jackson-modules-java8-datatypes and added buid_info.json and License.

### PR DESCRIPTION
**Pull request by santhoshkotla2929**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: j/jackson-modules-java8-datatypes/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+ Apache License
```
Comment (prachikurade1) [inline j/jackson-modules-java8-datatypes/LICENSE:1]: Fix indentation.

File: j/jackson-modules-java8-datatypes/jackson_modules-java8-datatypes_ubi_9.3.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : com.fasterxml.jackson.datatype:jackson-datatype-jdk8
+# Version       : 2.14.3
```
Comment (prachikurade1) [inline j/jackson-modules-java8-datatypes/jackson_modules-java8-datatypes_ubi_9.3.sh:5]: Match version as mentioned in build_info.json

File: j/jackson-modules-java8-datatypes/jackson_modules-java8-datatypes_ubi_9.3.sh [hunk 3]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : com.fasterxml.jackson.datatype:jackson-datatype-jdk8
```
Comment (prachikurade1) [inline j/jackson-modules-java8-datatypes/jackson_modules-java8-datatypes_ubi_9.3.sh:4]: Package name should match build_info.json

**Review by prachikurade1 — CHANGES_REQUESTED** [review]

**Review by prachikurade1 — APPROVED** [review]
Approved

File: j/jackson-modules-java8-datatypes/build_info.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "package_name": "jackson-modules-java8-datatypes",
+    "github_url": "https://github.com/FasterXML/jackson-modules-java8",
+    "version": "jackson-modules-java8-2.14.3",
+    "default_branch": "2.18",
+    "build_script": "jackson-modules-java8-datatypes_ubi_9.3.sh",
+    "package_dir": "j/jackson-modules-java8-datatypes",
+    "validate_build_script": true,
+    "use_non_root_user": false,
+    "docker_build": false,
+    "2.*.*": {
```
Comment (prachikurade1) [inline j/jackson-modules-java8-datatypes/build_info.json:12]: This version will be `jackson-modules-java8-2.*.*`

**Review by prachikurade1 — CHANGES_REQUESTED** [review]

**Review by prachikurade1 — APPROVED** [review]
Approved

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**